browser/brave_wallet/brave_wallet_service_factory.cc

```diff
@@ -72,6 +74,9 @@ BraveWalletServiceFactory::BraveWalletServiceFactory()
   DependsOn(JsonRpcServiceFactory::GetInstance());
   DependsOn(TxServiceFactory::GetInstance());
   DependsOn(BitcoinWalletServiceFactory::GetInstance());
+
+  ThirdPartyService::Get().SetLauncher(
+      std::make_unique<OutOfProcessThirdPartyServiceLauncher>());
```

Comment:
I need a place to specify the launcher before accessing these APIs in wallet component, I can move this if there's a better place for it.